demo/build.gradle

```diff
@@ -31,6 +31,7 @@ android {
     kotlinOptions {
         jvmTarget = "1.8"
         freeCompilerArgs = freeCompilerArgs +
+                "-Xopt-in=kotlin.RequiresOptIn" +
```

Comment:
Does the demo module need this?

Reply:
Yeap, there are file-wide opt-ins in examples